src/lib/cron.js

```diff
@@ -112,5 +113,7 @@ export default function runCronJobs() {
     weeklyJob.start();
     const monthlyJob = new CronJob(monthlySched, () => runMonthlyEmailJob(), null, true, timezone);
     monthlyJob.start();
+
+    const maintenanceJobs = runMaintenanceCronJobs(timezone);
```

Comment:
can remove assignment

Reply:
also, we should consider a job that purges old rows from MaintenanceLogs, otherwise the table will just grow forever